config/install/serialization.settings.yml

```diff
@@ -0,0 +1,2 @@
+bc_primitives_as_strings: false
+bc_timestamp_normalizer_unix: false
```

Comment:
I get an error from this config when installing the module:
`Configuration objects (rest.settings, serialization.settings) provided by dennis_term_manager already exist in active configuration`